### DISCUSSION TITLE
posts/arrays-javascript/

### DISCUSSION THREAD

**Discussion by giscus** [marcosruiz/marcosruiz.github.io#18] [General]
# posts/arrays-javascript/

Introducción

https://marcosruiz.github.io/posts/arrays-javascript/

**Comment by JessLar** [comment]
https://marcosruiz.github.io/posts/arrays-javascript/#trabajando-con-arrays

En el ejemplo que pones en esta parte con el método at(), ¿es correcto?

Es decir, at() accede y devuelve el elemento de un array, pero ¿podemos asignarle directamente un valor con el mismo método?

a.at(-1) = 2;
